buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java

```diff
@@ -500,20 +591,20 @@ public synchronized void start() {
             if (getVersion().onOrAfter("7.6.0")) {
                 logToProcessStdout("installing " + pluginsToInstall.size() + " plugins in a single transaction");
                 final String[] arguments = Stream.concat(Stream.of("install", "--batch"), pluginsToInstall.stream()).toArray(String[]::new);
-                runOpenSearchBinScript("opensearch-plugin", arguments);
+                runOpenSearchBinScript(currentConfig.pluginTool, arguments);
                 logToProcessStdout("installed plugins");
             } else {
                 logToProcessStdout("installing " + pluginsToInstall.size() + " plugins sequentially");
-                pluginsToInstall.forEach(plugin -> runOpenSearchBinScript("opensearch-plugin", "install", "--batch", plugin));
+                pluginsToInstall.forEach(plugin -> runOpenSearchBinScript(currentConfig.pluginTool, "install", "--batch", plugin));
                 logToProcessStdout("installed plugins");
             }
         }
```

Comment:
small change: do you want to move logToProcessStdout("installed plugins"); out side if else?

Reply:
Thanks! I did see many opportunities to refactor and clean up the code. While such changes are obvious, these are intentionally not included in this PR to keep the scope minimal.